PrebidMobile/DemoApp/src/main/java/org/prebid/mobile/demoapp/PrebidApplication.java

```diff
@@ -53,11 +53,14 @@ private void initialisePrebidForDFP() {
         adUnit1.addSize(320, 50);
 
         //Configure Ad-Slot2 with the same demand source
-        BannerAdUnit adUnit2 = new BannerAdUnit(DFP_BANNER_ADUNIT_300x250, "0c286d00-b3ee-4550-b15d-f71f8e746865");
+        BannerAdUnit adUnit2 = new BannerAdUnit(DFP_BANNER_ADUNIT_300x250, "eebc307d-7f76-45d6-a7a7-68985169b138");
         adUnit2.addSize(300, 250);
 
+        BannerAdUnit adUnit4 = new BannerAdUnit("random1", "eebc307d-7f76-45d6-a7a7-68985169b138");
```

Comment:
Same here and below `random1`and `Interstitial` need to be referenced from constants file